app/server/views/_index_table_of_contents.html.erb

```diff
@@ -0,0 +1,25 @@
+<section class = "l-collection-modules l-book-modules l-padding-60 l-block-wrapper l-block--1080">
```

Comment:
It's customary to not have spaces either side of the `=` in HTML tags.

Reply:
I've removed the space, I think